src/black/mode.py

```diff
@@ -197,6 +197,11 @@ class Preview(Enum):
     allow_form_feeds = auto()
 
 
+UNSTABLE_FEATURES: Set[Preview] = {
+    Preview.string_processing,
+}
```

Comment:
Would it be more prudent to instead have a set of "approved" preview features, and check that, rather than expecting someone to remember to add new features to this set before they're "ready" to include in the preview style?

Reply:
The intention is that new features go into the "approved" set by default (the approval is that we merge the PR). We'd put features in "unstable" only in the hopefully rare case where they are problematic in some way.